README.md

```diff
@@ -341,11 +345,26 @@ Translations of the guide are available in the following languages:
       puts 'Too long!'
     when Time.now.hour > 21
       puts "It's too late"
+    when :minus_op, :minus_minus_op
```

Comment:
Given this example, I think we're talking more about `Good` and `Better` than about `Bad` and `Good`.

Reply:
There's just one other precedent in the style guide.  I'm happy to make a simple `Better` block, and try to overall keep the examples 'deduped' to limit newlines.

Reply:
Updated @bbatsov.   A smaller diff now, too.